quantum-api/lib/file-transforms/changelog.js

```diff
@@ -213,7 +228,7 @@ function buildEntries (version, versions, tagSelections, entityTypeToLanguage) {
         res.push({
           type: 'entry',
           params: [],
-          content: [header].concat(tagSelections.map(tag => {
+          content: [header].concat(tagsForApi.map(tag => {
```

Comment:
This was the bug causing memory issues in the changelog

We were passing `tagSelections` (all the tags) instead of the tags for the current API to all sections which quickly ran out of memory when attempting to process the entire hexagon changelog